Modules/SubModule_Import/xASL_imp_NII2BIDS_SubjectSessionRun.m

```diff
@@ -298,6 +317,27 @@
             end
         end
     end
+    
+    
+    %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    % Check for FME Hadamard sequences
+    if isfield(jsonLocal,'SeriesDescription')
+        isHadamardFME = ~isempty(regexp(char(jsonLocal.SeriesDescription),'(Encoded_Images_Had)\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once'));
+        if isHadamardFME
+            startDetails = regexp(char(jsonLocal.SeriesDescription),'\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once');
+            HadamardType = xASL_str2num(jsonLocal.SeriesDescription(startDetails:startDetails+1),'auto');
+            HadamardNumberTI = xASL_str2num(jsonLocal.SeriesDescription(startDetails+3:startDetails+4),'auto');
```

Comment:
@jan-petr Should `Number` be `value`? Otherwise `Number` could be confused with `Count`